lib/errors.js

```diff
@@ -35,12 +35,14 @@ function toGraphQLError (err) {
 
   const gqlError = new GraphQLError(
     err.message,
-    err.nodes,
-    err.source,
-    err.positions,
-    err.path,
-    err,
-    err.extensions
+    {
+      nodes: err.nodes,
+      source: err.source,
+      positions: err.positions,
+      path: err.path,
+      originalError: err,
+      extensions: err.extensions
+    }
```

Comment:
why this change?

Reply:
That's because positional arguments in GraphQLError were deprecated but `graphql@16` was still supporting them.
And `graphql@17` drops their support in favour of object argument - https://github.com/graphql/graphql-js/pull/3577

Reply:
Yup! `16.0` that Mercurius uses is already having this code that migrates the positional args into a new signature if needed
<img width="614" alt="Screenshot 2022-10-26 at 11 46 51" src="https://user-images.githubusercontent.com/36277508/198007089-47d6ebb5-8ebb-4226-9753-c63da2b91742.png">

Reply:
awesome!